demos/ASP.Net MVC/Divergent.Frontend/Views/Orders/Details.cshtml

```diff
@@ -16,12 +15,3 @@
         <div>Courier: @Model.ShippingCourier</div>
     </div>
 </div>
-
-@*<div class="panel panel-default">
-    <div class="panel-heading">
-        <h3 class="panel-title">Shipment information</h3>
-    </div>
-    <div class="panel-body">
-
-    </div>
-</div>*@
```

Comment:
`.suo` files are there for the only purpose of sharing startup projects settings, so that we can provide a `F5` experience. I hate to be forced to commit that stuff.

Any other idea?